### ISSUE TITLE
🚀 Feature: <title>

### ISSUE THREAD

**Issue by DavidAn830** [backstage/backstage#17459]
### 🔖 Feature description

Our org is trying to add a filter and buttons to `ScaffolderPage`. However, customizing on this page is very limited at the moment. 

I would like to contribute to this component so that it takes custom components as well. 

### 🎤 Context

In our use case, we have created a button to export all the entities into a spreadsheet file. We have this button available in catalog pages but not in the scaffolder page. We would like to add this feature to the scaffolder page for our users. 

We would also like to add a custom picker / filter to the scaffolder page. 

These are currently blocked as customizing in the scaffolder page is limited. 

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by awanlin** [comment]
Hi @DavidAn830, just wanted to let know that there is an upcoming `scaffolder/next` version being actively tested at this stage which is a redesign of the scaffolder. The documentation about this is here: https://backstage.io/docs/features/software-templates/testing-scaffolder-alpha. Being able to provide your own component in places or completely is part of this redesign.

Related is this RFC that is looking for the same customization abilities: https://github.com/backstage/backstage/issues/17417

**Comment by DavidAn830** [comment]
@awanlin Thanks for that information! Do you know when `scaffolder/next` is going to be up and running?

**Comment by awanlin** [comment]
Yeah, I don't know the timeline for this @DavidAn830 but what's currently available is pretty stable in my experience. I highly recommend you determine that yourself by testing it out. You can use it behind a feature flag to help slowly roll it out or test with real world examples
